test/TestRRUtils.sol

```diff
@@ -99,4 +99,11 @@ contract TestRRUtils {
     require(bthLabXyz.compareNames(xyz)       >  0, "bthLab.xyz comes after xyz");
   }
 
+  function testSerialNumberGt() public pure {
+    require(RRUtils.serialNumberGt(1, 0), "1 > 0");
+    require(!RRUtils.serialNumberGt(0, 1), "!(0 < 1)");
+    require(RRUtils.serialNumberGt(0, 0xFFFFFFFF), "0 > 0xFFFFFFFF");
+    require(!RRUtils.serialNumberGt(0xFFFFFFFF, 0), "!(0 < 0xFFFFFFFF)");
+    require(RRUtils.serialNumberGt(0x11111111, 0xAAAAAAAA), "0x11111111 > 0xAAAAAAAA");
+  }
```

Comment:
Also worth adding boundary test taken from https://en.wikipedia.org/wiki/Serial_number_arithmetic
```
unsigned    binary     signed
sequence    value      distance
--------    ------     --------
   32767 == 0x7FFF ==  32767
       1 == 0x0001 ==      1
       0 == 0x0000 ==      0
   65535 == 0xFFFF ==     −1 
   65534 == 0xFFFE ==     −2
   32768 == 0x8000 == −32768
```

Reply:
Should still have a test case for `==` though